### ISSUE TITLE
Update to latest Cluster API version

### ISSUE THREAD

**Issue by mbardelmeijer** [CyberfusionIO/cyberfusion-cluster-api-client#68]
Specifically, we would like to use the `setOptimizingEnabled` & `setBackupsEnabled` attributes on the `Database` object. Added with https://cluster-api.cyberfusion.nl/redoc#section/Changelog/1.185.5-2023-07-22

**Comment by mbardelmeijer** [comment]
Makes sense! No problem at all, just wanted to create the issue to document it and track progress. Thanks for your continued work on this 💪

**Comment by dvdheiden** [comment]
Just released 1.101.0 which should be completely up-to-date. Feel free to open an issue when encountering any issues or the cluster API gets updated again 😄
